### PR TITLE
Install udev rule

### PR DESCRIPTION
**Pull request by jnippula**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -55,3 +55,8 @@ set_property(TARGET depthai_ctrl PROPERTY CXX_STANDARD_REQUIRED ON)
 set_property(TARGET depthai_ctrl PROPERTY CXX_EXTENSIONS OFF)
 
 install(TARGETS depthai_ctrl)
+
+install(FILES
```
Comment (maseabunikie) [inline CMakeLists.txt:59]: Do we need to verify that the permissions are ok for the 80-movidius.rules file?

File: 80-movidius.rules [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#SUBSYSTEM=="usb", ATTRS{idVendor}=="03e7", ATTRS{idProduct}=="2485", TAG+="uaccess", TAG+="systemd", ENV{SYSTEMD_WANTS}="depthai_ctrl.service"
+
+#SUBSYSTEM=="usb", ATTRS{idVendor}=="03e7", ATTRS{idProduct}=="2485", MODE="0666",  GROUP="plugdev"
+
+SUBSYSTEM=="usb", ATTRS{idVendor}=="03e7", TAG+="uaccess", TAG+="systemd", ENV{SYSTEMD_WANTS}="depthai_ctrl.service"
```
Comment (maseabunikie) [inline 80-movidius.rules:6]: This can be commented for the moment. It has not been properly tested.

File: CMakeLists.txt (hunk 1 above)
Comment (jnippula) [reply CMakeLists.txt:59]: Done

File: 80-movidius.rules [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+#SUBSYSTEM=="usb", ATTRS{idVendor}=="03e7", ATTRS{idProduct}=="2485", TAG+="uaccess", TAG+="systemd", ENV{SYSTEMD_WANTS}="depthai_ctrl.service"
+
+#SUBSYSTEM=="usb", ATTRS{idVendor}=="03e7", ATTRS{idProduct}=="2485", MODE="0666",  GROUP="plugdev"
+
+#SUBSYSTEM=="usb", ATTRS{idVendor}=="03e7", TAG+="uaccess", TAG+="systemd", ENV{SYSTEMD_WANTS}="depthai_ctrl.service"
+SUBSYSTEM=="usb", ATTRS{idVendor}=="03e7", MODE="0666"
```
Comment (jnippula) [inline 80-movidius.rules:7]: Does it look correct now?

Comment (maseabunikie) [reply 80-movidius.rules:7]: It is correct now.

**Review by maseabunikie — APPROVED** [review]

**Review by maseabunikie — APPROVED** [review]

**Pull request merged**